webi-essentials/README.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Webi Essentials
+homepage: https://webinstall.dev/webi-essentials
+tagline: |
+  The Webi Essentials are curl, git, tar, wget, xz, zip, & webi shell integrations
+---
+
+### Cheat Sheet
+
+> Even Webi needs a place to start.
+
+This installs the tools that are commonly needed to download and unpack various
+packages, using the native package manager.
+
+**This requires root or `sudo`**
```

Comment:
May be better to add something about Administrator access, being as this package is cross-platform? Windows doesn't natively have a concept of `sudo` or even `root`.

Reply:
This doesn't require root on windows and it does ask if you want to use sudo before it does anything.